app/models/document_type_selection/option.rb

```diff
@@ -18,4 +18,8 @@ def managed_elsewhere?
   def managed_elsewhere_url
     hostname ? Plek.new.external_url_for(hostname) + path : path
   end
+
+  def pre_release?
+    DocumentType.find(id).pre_release? if document_type?
```

Comment:
Nice job 👍 

Since this method should only return a bool should we do either:

```
return false unless document_type?
DocumentType.find(id).pre_release?
```

or:

```
document_type? ? DocumentType.find(id).pre_release? : false
```